testsuite/tests/kuadrant/authorino/wristband/test_wristband.py

```diff
@@ -0,0 +1,30 @@
+"""Edge Authentication tests"""
+from cryptography.x509 import load_pem_x509_certificate
+
+import jwt
+
+
+def test_wristband_token_claims(wristband_token, wristband_endpoint, certificates):
+    """Verify acquired jwt token claims"""
+    # extract public key from the certificate
+    cert_encoded = certificates["signing_ca"].certificate.encode("utf-8")
+    cert_object = load_pem_x509_certificate(cert_encoded)
+    public_key = cert_object.public_key()
+    # decode jwt to examine claims
+    wristband_decoded = jwt.decode(wristband_token, key=public_key, algorithms=["RS256"])
+    assert wristband_decoded["exp"] - wristband_decoded["iat"] == 300
+    assert wristband_decoded["iss"] == wristband_endpoint
```

Comment:
Could you also test that the token contains less information than the original one from RHSSO?